pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -844,6 +844,10 @@ type NodeGroup struct {
 	// +optional
 	Taints taintsWrapper `json:"taints,omitempty"`
 
+	// UpdateConfig configures how to update NodeGroups.
+	// +optional
+	UpdateConfig *NodeGroupUpdateConfig `json:"updateConfig,omitempty"`
```

Comment:
Would this be better named `UpgradeConfig` given the update vs upgrade mess we have had in the past?

Reply:
Very valid point, that logic/naming would definitely make more sense for `eksctl`, however the AWS docs will be calling this new field `UpdateConfig`, so it might be better to keep it like this for consistency despite the slight cognitive dissonance?

Reply:
omigod they suck. uuuugggh really hate calling this update